src/modules/sharing/sharing.controller.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { SharingController } from './sharing.controller';
+import { SharingService } from './sharing.service';
+import { UuidDto } from '../../common/uuid.dto';
+import { SharingTypeDto } from './dto/sharing-type.dto';
+import { createMock } from '@golevelup/ts-jest';
+
+describe('SharingController', () => {
+  let controller: SharingController;
+  let sharingService: SharingService;
+
+  beforeEach(async () => {
+    sharingService = createMock<SharingService>();
+    controller = new SharingController(sharingService);
+  });
+
+  describe('get sharing size', () => {
+    it('When request the get sharing size method, then it works', async () => {
+      const sharingId = 'valid-sharing-id';
```

Comment:
let's use `chance` library to generate a valid sharing id (a uuid) so we validate that the format control is working properly. This should be already done if you export the fixture for sharing and you use it for generating a new one